common/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="TypographyEllipsis">
+    <string name="pool_item_subtitle_apy">%s / per year</string>
```

Comment:
We should clarify it with Andrey, there seems to be a typo - `/` already means `per`